build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -1,11 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\..\Configuration.props" />
-  <Import Project="mono-runtimes.props" />
-  <Import Project="mono-runtimes.projitems" />
   <PropertyGroup>
-    <_MonoPath>..\..\external\mono</_MonoPath>
+    <_SourceTopDir>..\..</_SourceTopDir>
+    <_BclFrameworkDir>$(OutputPath)\lib\xbuild-frameworks\MonoAndroid\v1.0</_BclFrameworkDir>
+    <_MonoPath>$(_SourceTopDir)\external\mono</_MonoPath>
```

Comment:
Why introduce and use `$(_SourceTopDir)`?

Related: as part of monodroid integration, we *may* want to create a "toplevel" `$(MonoPath)` property that is the path to the Mono checkout to use, in lieu of using `git submodule update`, along with Java.Interop, so that we can bump mono/etc. without a `xamarin-android` commit. (Useful for a "rebuild xamarin-android with mono/master..." lane.)

As such, we may want to just do that now, possibly by altering `Condfiguration.prop` with:

    <MonoPath Condition=" '$(MonoPath)' != '' ">$([System.IO.Path]::GetFullPath('external/mono`))</MonoPath>

so that we have an absolute path.